pkgs/development/compilers/dotnet/update.sh

```diff
@@ -390,17 +388,33 @@ Examples:
   runtime_$major_minor_underscore = buildNetRuntime {
     version = \"${runtime_version}\";
     $runtime_sources
-  };
-
-  sdk_$major_minor_underscore = buildNetSdk {
+  };" > "${result}"
+
+    declare -A feature_bands
+    unset latest_sdk
+
+    for sdk_version in "${sdk_versions[@]}"; do
+      sdk_base_version=${sdk_version%-*}
+      feature_band=${sdk_base_version:0:-2}xx
+      # sometimes one release has e.g. both 8.0.202 and 8.0.203
+      [[ ! ${feature_bands[$feature_band]+true} ]] || continue
```

Comment:
Do we have a guarantee here that they're sorted in highest to lowest version? Or do we risk ending up with 8.0.202 in nixpkgs when 8.0.203 was available but placed later on the list of SDK versions?

Reply:
Yeah, it sorts them descending and then takes the first one for each feature level.

I'm not sure why both of those sdks are included in the 2.0.3 'release'.